app/src/main/java/com/nononsenseapps/feeder/archmodel/FeedItemStore.kt

```diff
@@ -143,6 +143,12 @@ class FeedItemStore(
         }
 
         when {
+            filter.search.isNotEmpty() -> {
+                append("AND (\n")
+                append("plain_title LIKE ?\n").also { args.add("%${filter.search}%") }
```

Comment:
So this will be problematic if the user searches for something with a percent sign. Like `50%`.

Something like this should be done:

```
val sanitizedSearch = filter.search
    // User probably means a literal backslash
    .replace("\\", "\\\\")
    // User probably means a literal percent sign
    .replace("%", "\\%")
    // User probably means a literal underscore
    .replace("_", "\\_")
```

And then also specify escape for the like, e.g.

```
append("plain_title LIKE ? ESCAPE '\\'\n").also { args.add("%$sanitizedSearch%") }
```

Same for next LIKE as well.

I realize this is an issue for the tag as well, but I think it's less of a problem there (but feel free to santize that too!)